### ISSUE TITLE
Add Blind Accessibility with Aria

### ISSUE THREAD

**Issue by ahicks92** [gabrielecirulli/2048#129]
I'm going to look at doing this.  I need an intro project for Aria, which is the new HTML5 accessibility standard.  It is technically possible to make 2048 accessible with Firefox and NVDA at least, but I do not yet know how much work it would be.  A friend of mine thinks that it should be simple.  Other screen readers do not implement the full Aria spec, so may not work-but the problem would be on their end, and I can't do much about it.
The one difficulty is as follows: in order to review the game sensibly, it must be tagged as a table.  The default screen reader table navigation commands are ctrl+alt+the arrow keys; this is clearly awkward, and could lead to possible moves.  Thus, as part of this process, an invisible cursor for exploration of the board should probably be added, which can announce the current tile's value via an Aria Live Region.  The second motivation for this cursor is that, in order to get screen readers to stop blocking the arrow keys without expecting their users to know advanced tricks, the game probably needs to be given roll Application.  The keys I nominate for this nonvisual cursor are WASD, but this is perhaps not the most logical combination for a visually impaired audience who lacks experience with games that use them for movement.
This should constitute no visual change and require no style sheet editing.  The added enhancement/feature will be visible only to those with screen readers, as Aria has no impact on sighted viewers.
I am visually impaired myself.  This can be played, if it is made accessible; blind people have implementations of Minesweeper and other games that use much larger grids than 4x4.
I'm posting here so that others with an interest in such accessibility can comment and make suggestions; I will submit a pull request when/if I have complete the work.
